### PR TITLE
luci-mod-system: add zram options

### PR DESCRIPTION
**Pull request by Ansuel**
This adds option to configure zram size and compression algo
@jow- 
Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

File: modules/luci-mod-system/luasrc/model/cbi/admin_system/system.lua [hunk 1]
```diff
@@ -105,6 +106,29 @@ o:value(8, translate("Normal"))
 o:value(9, translate("Warning"))
 
 
+--
+-- Zram Properties
+--
+if has_zram then
+	o = s:taboption("zram", Value, "zram_size_mb", translate("ZRam Size"), "Mb")
```
Comment (jow-) [inline modules/luci-mod-system/luasrc/model/cbi/admin_system/system.lua]: Use "MB" please (or is it indeed Mega*bit*? In this case abbreviate it as `MBit`)

Also given that this is the description string, maybe be a bit more verbose, like `Size of the ZRam device in megabytes`

File: modules/luci-mod-system/luasrc/model/cbi/admin_system/system.lua [hunk 2]
```diff
@@ -105,6 +106,29 @@ o:value(8, translate("Normal"))
...
+	o.datatype    = "uinteger"
+	
+	o = s:taboption("zram", ListValue, "zram_comp_algo", translate("ZRam Compression Algorithm"))
+	o.optional    = true
+	o.placeholder = lzo
+	o:value("lzo", "lzo")
+	o:value("lz4", "lz4")
+	o:value("deflate", "deflate")
+	
+	o = s:taboption("zram", Value, "zram_comp_streams", translate("ZRam Compression Streams"), translate("Number of core to be used for compression"))
```
Comment (jow-) [inline modules/luci-mod-system/luasrc/model/cbi/admin_system/system.lua]: Alternative suggestion for the help text:
`Number of parallel threads used for compression`

**Comment by Ansuel** [comment]
@jow- done ;)

**Comment by jow-** [comment]
Merged, thanks

**Pull request merged**